toqito/matrices/entangled_subspace.py

```diff
@@ -0,0 +1,119 @@
+"""Produces a basis of an r-entangled subspace."""
```

Comment:
make sure you add your function to the init file in `toqito/matrices`. 

Right now, your function can't be imported i.e. there will be pytest and doctest failures if I approve the workflow run.